mache-core/src/main/java/com/excelian/mache/core/SchemaOptions.java

```diff
@@ -1,5 +1,9 @@
 package com.excelian.mache.core;
 
+/**
+ * Enum to denote what should be done with the schema created in the underlying
+ * datastore on startup and shutdown.
+ */
 public enum SchemaOptions {
     CREATESCHEMAIFNEEDED(true, false),
```

Comment:
I'd prefer underscores TBH, but I didn't want to weigh in with my chainsaw too early!